jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/URIUtilTest.java

```diff
@@ -1045,7 +1021,7 @@ public void testEncodeDecodeVisibleOnly()
         {
             assertTrue(c > 0x20 && c < 0x80);
```

Comment:
Should not `c < 0x7F`, since `7F` is encoded and can never be present as a character.
Also, the comment 2 lines above has a typo: `endoded`.

Reply:
fair enough, but the check for control characters is checked 2 lines below this, which would have tripped on the raw 0x7f usage as well.